frontend/views/pages/Page.vue

```diff
@@ -23,6 +23,7 @@ div(:data-test='pageTestName' :class='$scopedSlots.sidebar ? "p-with-sidebar" :
     v-if='$scopedSlots.sidebar'
     :class='{ "is-active": ephemeral.isActive }'
   )
+    i18n.sr-only(tag='h2') Page Details
```

Comment:
Does this need a `.is-title-2` on it? If not, can you add a comment explaining why not? (And maybe add a comment to #805 to use this as an example of when not to use it?)

Reply:
YEs, to stay consistent we need the class to the title now

Reply:
No, it doesn't. This is a heading only for screen readers, so they know what's this sidebar about. That's the purpose of `.sr-only`.